code/modules/gear_presets/upp.dm

```diff
@@ -107,6 +107,7 @@
 /datum/equipment_preset/upp/soldier/dressed
 	name = "UPP Soldier"
 	flags = EQUIPMENT_PRESET_EXTRA
+	skills = /datum/skills/upp
```

Comment:
is this supposed to be different to `/datum/equipment_preset/upp/soldier`?

Reply:
yes, the dressed versions are the ones used by ERTs and are ment to have the extra skills